pkg/apis/core/types.go

```diff
@@ -2441,6 +2441,13 @@ type ResourceClaim struct {
 	// the Pod where this field is used. It makes that resource available
 	// inside a container.
 	Name string
+
+	// Request is the name chosen for a request in the referenced claim.
+	// If empty, everything from the claim is made available, otherwise
+	// only the result of this request.
+	//
+	// +optional
+	Request string
```

Comment:
In the KEP we called this `RequestName`. I don't feel strongly, just making sure this was reviewed / requested to be changed, rather than an oversight.

Reply:
Yes. Tim pointed out that `<type>Name` is used for API resources, while this is a reference within the resource. I explicitly asked and he agreed with this change.